libs/ha/ha_common_libs_k8s.py

```diff
@@ -587,7 +586,7 @@ def partial_multipart_upload(self, s3_data, bucket_name, object_name, part_numbe
                 parts_etag.append({"PartNumber": part, "ETag": p_etag["ETag"]})
                 LOGGER.info("Uploaded part %s", part)
             return True, mpu_id, multipart_obj_path, parts_etag
-        except BaseException as error:
+        except CTException as error:
```

Comment:
There are too many operations happening in try block, and all are not raising CTException on failure.
So I had added BaseException to catch generic failure also. Please revert the change.